src/sentry/assistant/guides.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+
+GUIDES = {
+    'issue': {
+        'id': 1,
+        'page': 'issue',
+        'cue': 'Click here for a tour of the issue page',
```

Comment:
It's a shame we won't get i18n on all this text since it's all returned by the API

Reply:
That's not true.

```python
from django.utils.translation import ugettext_lazy as _
...

'cue': _('Click here for a tour of the issue page'),
```

Reply:
👍